### PR TITLE
Remove references to obsolete project.

### PR DESCRIPTION
**Pull request by davorg**
I saw someone linking to this page on StackOverflow and realised that it still recommended Padre (which has been dead since 2013). So I've done a quick clean-up.

### CODE REVIEW

File: docs/learn/tpl/defaults [hunk 1]
```diff
@@ -16,6 +16,9 @@
```
Comment (rspier) [inline docs/learn/tpl/defaults:16]: Line 15 can probably be deleted.

File: docs/learn/tpl/defaults [hunk 2]
```diff
@@ -16,6 +16,9 @@
 
     SET strawberry_padre_perl_version = '5.12.3';
```
Comment (rspier) [inline docs/learn/tpl/defaults:17]: And this line.

File: docs/learn/tpl/defaults [hunk 3]
```diff
@@ -16,6 +16,9 @@
...
 
+    SET strawberry = 'https://strawberryperl.com/';
```
Comment (rspier) [inline docs/learn/tpl/defaults:19]: This is unlikely to ever change,  it probably doesn't need to be a variable.

**Review by rspier — CHANGES_REQUESTED** [review]
Thanks!

**Comment by davorg** [comment]
Thanks. I've made the suggested changed.

**Review by rspier — APPROVED** [review]

**Comment by rspier** [comment]
Thanks!

**Pull request merged**